Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -118,27 +118,6 @@ private set
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="IncrementalLoadingCollection{TSource, IType}"/> class optionally specifying how many items to load for each data page.
-        /// </summary>
-        /// <param name="itemsPerPage">
-        /// The number of items to retrieve for each call. Default is 20.
-        /// </param>
-        /// <param name="onStartLoading">
-        /// An <see cref="Action"/> that is called when a retrieval operation begins.
-        /// </param>
-        /// <param name="onEndLoading">
-        /// An <see cref="Action"/> that is called when a retrieval operation ends.
-        /// </param>
-        /// <param name="onError">
-        /// An <see cref="Action"/> that is called if an error occours during data retrieval.
-        /// </param>
-        /// <seealso cref="IIncrementalSource{TSource}"/>
-        public IncrementalLoadingCollection(int itemsPerPage = 20, Action onStartLoading = null, Action onEndLoading = null, Action<Exception> onError = null)
-            : this(new TSource(), itemsPerPage, onStartLoading, onEndLoading, onError)
```

Comment:
Removing this method is a breaking change. Existing code will fail with the new version. It should still allow for the source to be generated for the user.